docs/server/source/drivers-clients/index.rst

```diff
@@ -1,10 +1,18 @@
 Drivers & Clients
 =================
 
+Currently, the only language-native driver is written in the Python language.
+
+We also provide the Transaction CLI to be able to script the building of
+transactions. You may be able to wrap this tool inside the language of
+your choice, and then use the server api directly to post transactions.
+
+
 .. toctree::
    :maxdepth: 1
 
    http-client-server-api
    python-driver
```

Comment:
@ttmc With these changes, I wonder if we should just link to the python driver's docs here instead of having a different page that does that?

We can handle this separately.